testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -460,35 +474,6 @@ public void testMaterializedView()
         assertUpdate("DROP MATERIALIZED VIEW " + viewName);
     }
 
-    // TODO inline when all implementations fixed
-    protected void checkShowColumnsForMaterializedView(String viewName)
```

Comment:
it could be good to have a commit that does the inline, separate from other changes